scripts/utils/exec.js

```diff
@@ -2,7 +2,7 @@
  * Borrowed from React Bootstrap
  * https://github.com/react-bootstrap/react-bootstrap
  */
-const processExecSync = require('child_process').execSync;
+const {shellSync} = require('execa');
```

Comment:
We were using the `child_process` tool, because bootstrap was using it at the time, but they've moved on to `execa` and I'm just following suit